File: codeforces/21.translation/translation.cpp
#include <iostream>
#include <cstring>
/* Author: José Rodolfo (jric2002) */
using namespace std;
int main() {
  string s, t;
  unsigned short int size_s, number_equal_char = 0;
  bool is_correct = true;
  cin >> s;
  cin >> t;
  size_s = s.size();
  if (size_s == t.size()) {
    for (unsigned short int i = 0, j = size_s - 1; i < size_s; i++, j--) {
      if (s[i] != t[j]) {
        is_correct = false;
        break;
      }
    }
  }
  else {
    is_correct = false;
  }
  if (is_correct) {
    cout << "YES" << endl;
  }
  else {
    cout << "NO" << endl;
  }
  return 0;
}

File: codeforces/21.translation_[41A]/translation.py
# Author: José Rodolfo (jric2002)
s = str(input())
t = str(input())
is_correct = True
size_s = len(s)
size_t = len(t)
i = 0
j = size_s - 1
if (size_s == size_t):
  while (i < size_s):
    if (s[i] != t[j]):
      is_correct = False
      break
    i += 1
    j -= 1
else:
  is_correct = False
print("YES") if (is_correct) else print("NO")